sequencer.c

```diff
@@ -766,6 +766,32 @@ N_("you have staged changes in your working tree\n"
 #define VERIFY_MSG  (1<<4)
 #define CREATE_ROOT_COMMIT (1<<5)
 
+static int run_command_silent(struct child_process *cmd,
+			      unsigned silent)
+{
+	if (silent) {
```

Comment:
Maybe you could just `return run_command(cmd);` in the !silent case first. This would decrease indentation in the rest of the function.